server/src/main/java/org/spine3/server/Entity.java

```diff
@@ -50,6 +59,8 @@
 
     private int version;
 
+    private final M defaultState;
```

Comment:
Have map per class. We cannot have default state in every instance. It is memory-expensive.